config/_ytt_lib/github.com/cloudfoundry/capi-k8s-release/templates/ccdb-migrate-job.yaml

```diff
@@ -0,0 +1,26 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: ccdb-migrate
+  namespace: #@ data.values.system_namespace
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
```

Comment:
Doesn't this annotation imply that this pod won't have an istio sidecar and therefore will not do istio-mediated mtls with the database pod?

Reply:
it looks like this branch was not merged before the pr was opened: https://github.com/cloudfoundry/capi-k8s-release/compare/db-migration-with-sidecars

Reply:
this is almost certainly why the `concourse-ci/tests` check is failing